shoryuken.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.6'
   spec.add_development_dependency 'rake'
   spec.add_development_dependency 'rspec'
-  spec.add_development_dependency 'pry-byebug'
+  spec.add_development_dependency 'pry-debugger'
```

Comment:
@geekguy hm this one breaks the compatibility with Ruby 2.0. If we keep `pry-byebug` as `add_development_dependency` can you still run Shoryuken with 1.9.3?